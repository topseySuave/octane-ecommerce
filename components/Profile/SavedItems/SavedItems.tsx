import { Typography } from "antd";
import { ProductCardHorizontal } from 'components/ui/ProductCard';

const { Title } = Typography;

const SavedItems = () => {
  return (
    <>
      <Title level={3}>Saved Item(s)</Title>
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
      <ProductCardHorizontal />
    </>
  );
};

export default SavedItems;
