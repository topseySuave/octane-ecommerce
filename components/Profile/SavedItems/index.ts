import SavedItems from './SavedItems';

export default SavedItems;
