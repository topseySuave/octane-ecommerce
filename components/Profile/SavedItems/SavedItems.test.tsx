import { shallow } from 'enzyme';
import React from 'react';

import SavedItems from './SavedItems';

describe('SavedItems', () => {
  it('Renders with saved items', () => {
    const tree = shallow(<SavedItems />);
    expect(tree).toMatchSnapshot();
    // const mounted = shallow(<Header />);
    // const header = mounted.find('header');
    // console.log(header);
    // expect(header.hasClass('main-header-container')).toBeTruthy();
  });
});
