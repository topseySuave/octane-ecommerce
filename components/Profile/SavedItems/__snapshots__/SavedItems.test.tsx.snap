// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SavedItems Renders with saved items 1`] = `ShallowWrapper {}`;
